src/app/app.module.ts

```diff
@@ -7,16 +7,16 @@ import { HttpModule } from '@angular/http';
 import { AppRoutingModule } from './app.router';
 import { AppComponent } from './app.component';
 import { HeaderComponent } from './components/header/header.component';
-import { AllUsersTableComponent } from './components/all-users-table/all-users-table.component';
+import { AllUsersTableComponent } from './components/gamification/all-users-table/all-users-table.component';
 import { GamificationService } from './service/gamification.service';
 import { UserService } from './service/user.service';
-import { UserDetailsTableComponent } from './components/user-details-table/user-details-table.component';
+import { UserDetailsTableComponent } from './components/gamification/user-details-table/user-details-table.component';
 import { ScrollDirective } from './directives/scroll.directive';
 import { GoTopComponent } from './components/go-to-top/go-to-top.component';
-import { ActiveKeepersComponent } from './components/active-keepers/active-keepers.component';
+import { ActiveKeepersComponent } from './components/gamification/active-keepers/active-keepers.component';
 import { KeepersService } from './service/keepers.service';
 import { TeamsService } from './service/teams.service';
-import { TeamsComponent } from './components/teams/teams.component';
+import { TeamsComponent } from './components/gamification/teams/teams.component';
```

Comment:
I think teams is different service. 
So could you change root folder name to team

Reply:
Done